### ISSUE TITLE
Core dumped: Segmentation fault

### ISSUE THREAD

**Issue by 6api** [facebook/hhvm#5358]
when i upgrade hhvm to 3.7.0
my os: debian wheezy 64bit
how can i fix these
```
# 0  HPHP::PDORequestData::requestShutdown() at /usr/bin/hhvm:0
# 1  HPHP::ExecutionContext::onRequestShutdown() at /usr/bin/hhvm:0
# 2  HPHP::hphp_context_shutdown() at /usr/bin/hhvm:0
# 3  HPHP::HttpRequestHandler::executePHPRequest(HPHP::Transport*, HPHP::RequestURI&, HPHP::SourceRootInfo&, bool) at /usr/bin/hhvm:0
# 4  HPHP::HttpRequestHandler::handleRequest(HPHP::Transport*) at /usr/bin/hhvm:0
# 5  HPHP::ServerWorker<std::shared_ptr<HPHP::FastCGIJob>, HPHP::FastCGITransportTraits>::doJobImpl(std::shared_ptr<HPHP::FastCGIJob>, bool) at /usr/bin/hhvm:0
# 6  HPHP::ServerWorker<std::shared_ptr<HPHP::FastCGIJob>, HPHP::FastCGITransportTraits>::doJob(std::shared_ptr<HPHP::FastCGIJob>) at /usr/bin/hhvm:0
# 7  HPHP::JobQueueWorker<std::shared_ptr<HPHP::FastCGIJob>, HPHP::Server*, true, false, HPHP::JobQueueDropVMStack>::start() at /usr/bin/hhvm:0
# 8  HPHP::AsyncFuncImpl::ThreadFunc(void*) at /usr/bin/hhvm:0
# 9  HPHP::start_routine_wrapper(void*) at /usr/bin/hhvm:0
# 10 start_thread at /lib/x86_64-linux-gnu/libpthread.so.0:0
# 11 __clone at /lib/x86_64-linux-gnu/libc.so.6:0

PHP Stacktrace:
```

**Comment by paulbiss** [comment]
There are some known pdo bugs that were fixed recently. The patches have been cherry-picked into 3.7 but we haven't rolled a new package yet. You can try the nightly build (it has the fix too) or build the 3.7 branch yourself to see if your crash is resolved by the patch.

**Comment by Orvid** [comment]
Closing due to inactivity.

https://github.com/facebook/hhvm/wiki/Human-Timeouts
